### PR TITLE
LPD-37512 The end date is not displayed in the pop-up for events lasting more than one day

### PR DESCRIPTION
**Pull request by pedro-oliveira446**
[LPD-37512](https://liferay.atlassian.net/browse/LPD-37512)

[LPD-37512]: https://liferay.atlassian.net/browse/LPD-37512?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Comment by pedro-oliveira446** [comment]
ci:test:calendar

**Comment by pedro-oliveira446** [comment]
ci:test:sf

File: modules/apps/calendar/calendar-web/src/main/resources/META-INF/resources/js/legacy/scheduler_event_recorder.js [hunk 1]
```diff
@@ -660,6 +661,44 @@ AUI.add(
...
+						formattedEndDate = '';
+					}
+
+					const scheduler = event.get('scheduler');
+					const fmtHourFn = scheduler.get('activeView').get('isoTime')
+						? DateMath.toIsoTimeString
+						: DateMath.toUsTimeString;
+
+					return [
+						formattedStartDate.concat(','),
```
Comment (igor-franca) [inline modules/apps/calendar/calendar-web/src/main/resources/META-INF/resources/js/legacy/scheduler_event_recorder.js:694]: Hey @pedro-oliveira446, could you check if in the case of events that end on the same day, we will have an extra unwanted comma here?

**Review by igor-franca — COMMENTED** [review]
Hi @pedro-oliveira446, great work here. 

I have a suggestion about the organization of commits, perhaps you could create an initial commit stating that you are bringing in and overriding a bind method, and then another commit adding your customization to this function. This is just a detail, separating what would be a new logic from what was brought from the alloy.

This may mean that questions will not be asked about the existing alloy logic in the next review.

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

File: modules/apps/calendar/calendar-web/src/main/resources/META-INF/resources/js/legacy/scheduler_event_recorder.js (hunk 1 above)
Comment (pedro-oliveira446) [reply modules/apps/calendar/calendar-web/src/main/resources/META-INF/resources/js/legacy/scheduler_event_recorder.js:696]: The comma after the start date is used to separate it from the hour, so it is always necessary.

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Comment by pedro-oliveira446** [comment]
ci:test:calendar

**Review by igor-franca — APPROVED** [review]
LGTM!

**Comment by pedro-oliveira446** [comment]
ci:report:30729068

**Comment by pedro-oliveira446** [comment]
ci:forward:force

**Pull request closed without merging**